docs/user-guide/deploy-chart/examples/deploying-kube-prometheus-helm-chart.md

```diff
@@ -0,0 +1,55 @@
+# Deploying Kube-Prometheus Helm Chart
+
+## Introduction
+
+`bitnami/kube-prometheus` Helm chart bootstraps a single node kube-prometheus deployment on a Kubernetes cluster using the Helm package manager.
+
+![](../../../images/deploy-chart/examples/kube-prometheus/kube-prometheus.png)
+
+## 1. Discover Kube-Prometheus chart from Chart Store
+
+Select `Charts` from the left panel to visit the `Chart Store` page. You will see numerous of charts on the page from which you have to find `bitnami/kube-prometheus` chart. You also can use the search bar to search the MySQL chart.
+
+![](../../../images/deploy-chart/examples/kube-prometheus/kube-prometheussearch.png)
+
+## 2. Configure the Chart
+
+After selecting the `bitnami/kube-prometheus` Helm chart, click on `Deploy`.
+
+![](../../../images/deploy-chart/examples/kube-prometheus/kube-prometheusdeploy.png)
+
+Enter the following details, to deploy Kube-Prometheus chart:
+
+| Key | Description |
+| :--- | :--- |
+| `App Name` | Name of the Chart |
+| `Project` | Select the name of your Project in which you want to deploy the chart |
+| `Environment` | Select the environment in which you want to deploy the chart |
+| `Chart Version` | Select the latest Chart Version |
+| `Chart Value` | Select the default value or create a custom value |
+
+
+### Configure `values.yaml`
+
+You can configure the `values.yaml` according to your project's requirements.
+
+![](../../../images/deploy-chart/examples/kube-prometheus/kube-prometheusyaml.png)
+
+
+Click on `Deploy Chart` to deploy the Chart.
+
+## 3. Check the Status of Deployment
+
+After clicking on `Deploy` you will be redirected to app details page where you can see deployment status of the chart. The Status of the chart should be `Healthy`. It might take few seconds after initiating the deployment of the chart.
```

Comment:
After clicking `Deploy`, you will be redirected to `App Details` page where you can see deployment status of the chart. The Status of the chart must be `Healthy`. It might take few seconds after initiating the deployment of the chart.